### PR TITLE
Add nicer solution for unsupported market

### PR DESCRIPTION
**Pull request by pducks32**

### CODE REVIEW

**Comment by pducks32** [comment]
Hey don't push this, I'll add some tests

**Comment by kaplanmaxe** [comment]
Thanks! This looks good, I like the idea. Ignore travis for now. Just added tests today. I'll fix that in the mean time

**Comment by pducks32** [comment]
Cool cool. This is a neat project, I love command line tools! Have one other fix to this area of the code I'll add in this PR

**Comment by kaplanmaxe** [comment]
Thank you! I appreciate that. Just fixed travis to run correctly. When you recommit, it should run all the tests now, including eslint. Just pull master again into your branch and you will get the latest changes.

**Comment by pducks32** [comment]
Okay just kinda fixed it up because I was using GDAX and the output was kinda cryptic because Ripple for example wasn't supported. But CMC and GDAX were kinda different so I rather just have it print what is not supported.

**Comment by pducks32** [comment]
@kaplanmaxe this all good?

**Comment by kaplanmaxe** [comment]
Sorry, on vacation right now. Will check this out tomorrow. Thanks again!

File: src/classes/PortfolioResolver.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      };
+    });
+  }
+
+  /**
+   * Returns the market api that is requested.
+   *
+   * @return {object}
+   */
+  get currencyResolver() {
```
Comment (kaplanmaxe) [inline src/classes/PortfolioResolver.js:104]: Should this be one word?

Comment (kaplanmaxe) [reply src/classes/PortfolioResolver.js:104]: Sorry, wasn't thinking. Lack of sleep :)

**Pull request merged**